src/mame/drivers/kchamp.cpp

```diff
@@ -492,14 +492,14 @@ void kchamp_state::kchamp(machine_config &config)
 	GFXDECODE(config, m_gfxdecode, m_palette, gfx_kchamp);
 	PALETTE(config, m_palette, FUNC(kchamp_state::kchamp_palette), 256);
 
-	/* sound hardware */
+	// Sound hardware
 	SPEAKER(config, "speaker").front_center();
 
 	GENERIC_LATCH_8(config, m_soundlatch);
 	m_soundlatch->data_pending_callback().set_inputline(m_audiocpu, 0);
 
-	AY8910(config, m_ay[0], XTAL(12'000'000)/12).add_route(ALL_OUTPUTS, "speaker", 0.3); /* Guess based on actual pcb recordings of karatedo */
-	AY8910(config, m_ay[1], XTAL(12'000'000)/12).add_route(ALL_OUTPUTS, "speaker", 0.3); /* Guess based on actual pcb recordings of karatedo */
+	AY8910(config, m_ay[0], XTAL(12'000'000)/12).add_route(ALL_OUTPUTS, "speaker", 0.3); // Guess based on actual pcb recordings of karatedo
+	AY8910(config, m_ay[1], XTAL(12'000'000)/12).add_route(ALL_OUTPUTS, "speaker", 0.3); // Guess based on actual pcb recordings of karatedo
```

Comment:
Why didn’t you capitalise “PCB” in these comments when you changed the lines?

Reply:
Nah, it's OK, I just didn't see them.

Done. Thanks!